### ISSUE TITLE
WPML plugin issue - Documented fix

### ISSUE THREAD

**Issue by TonyKnibb-MakaraHealth** [pantheon-systems/documentation#9362]
I think these docs are incorrect: https://docs.pantheon.io/plugins-known-issues#wpml---the-wordpress-multilingual-plugin

## Expected behavior
.mo files are created in the `languages/wpml` directory

## Actual behavior
.mo files are created in a `languages/wpml/wpml` directory

### Steps to reproduce the problem
Following the docs results in the creation of a `wpml` directoryinside the `wpml` directory, and an unhappy website.
`define( 'WP_LANG_DIR','/files/languages/wpml' );`

## Suggested Resolution
Looking at the existing directorystructure that the plugin has created, suggests that the correct setting would be:
`define( 'WP_LANG_DIR','/files/languages' );`

## Additional information
Is it also worth noting in the docs that, when connecting using SFTP via the Windows 11 Terminal, you can use `ln -s uploads/languages languages` to create the necessary symlink, assuming you're in the `/code/wp-content` directory?

**Comment by rachelwhitton** [comment]
We're currently suggesting that you have to create a symlink AND adjust the `WP_LANG_DIR` constant. I suspect only one of these is needed, not both
* If you don't adjust the `WP_LANG_DIR` constant then you can fix via symlink to the uploads dir 
* If you don't symlink, you can fix by setting `WP_LANG_DIR` to `$_SERVER['HOME'] .'/files/languages'` 

Either way, you'd still need to manually create the `./files/languages` and `./files/languages/wpml` directories on Pantheon via SFTP. Then update translations, and check folder contents - we should see .mo files generated in that path now 

You shouldn't have to set `WP_TEMP_DIR`, that's handled by `wp-config-pantheon.php` ([source](https://github.com/pantheon-systems/WordPress/blob/master/wp-config-pantheon.php#L68-L69))
You shouldn't have to set `FS_METHOD` to direct, that's already handled by the Pantheon MU plugin ([source](https://github.com/pantheon-systems/WordPress/blob/master/wp-content/mu-plugins/pantheon-mu-plugin/pantheon.php#L31-L41))

---

We should test and validate the above and if true, remove the symlink suggestions and align with the [WPML recommendations here](https://wpml.org/errata/hosting-with-read-only-folder-wp-content-languages-issues-with-wpml-string-translation/) 

@stevector @jazzsequence do either of you have a WPML license we can use to validate the above? 

@TonyKnibb-MakaraHealth if you have interest and time to help us validate the above, please do and let us know your results. We'll work on this issue either way 🙂 Thanks for reporting

**Comment by stevector** [comment]
I do not

**Comment by miriamgoldman** [comment]
@rachelwhitton I agree on removing the symlink. IMHO, we should probably change the order of implementation. Create the `/languages/` folder _first_ in files, and then add the constant (without the /wpml/ in the location). (No need for the $_SERVER portion.)

I tested this out to confirm and everything saves properly. So, ✅ to create a PR for this (happy to do so, lmk about the order of implementation)
